packages/create-discord-bot/template/JavaScript/package.json

```diff
@@ -11,17 +11,17 @@
 		"deploy": "node --require dotenv/config src/util/deploy.js"
 	},
 	"dependencies": {
-		"@discordjs/core": "^2.1.1",
-		"discord.js": "^14.20.0",
-		"dotenv": "^16.4.5"
+		"@discordjs/core": "^2.2.0",
+		"discord.js": "^14.21.0",
+		"dotenv": "^17.0.1"
```

Comment:
Didn't we want to remove this dependency and use `--env-file` instead?

Reply:
Even if we did, its out of scope for this PR and should be done in a separate one please